docs/modules/ROOT/pages/access-control.adoc

```diff
@@ -202,3 +134,132 @@ TIP: A recommended configuration is to grant both roles to a secure governance c
+const EXECUTION_DELAY = 5 * HOUR;
+    
+await manager.setGrantDelay(MINTER, GRANT_DELAY, { from: initialAdmin });
+await manager.grantRole(MINTER, EXECUTION_DELAY, { from: initialAdmin });
+```
+
+You may notice roles do not define a name. Opposed to the xref:api:access.adoc#AccessControl[`AccessControl`] case, roles aren't hardcoded in the contract as `bytes32` values but identified as numeric values. However, role labeling allows tooling discoverability for easier role exploration and it can be achieved using the xref:api:access.adoc#AccessManager-labelRole-uint64-string-[`labelRole`] function.
+
+```javascript
+await manager.labelRole(MINTER, "MINTER");
+```
+
+It's recommended to keep only a single admin address secured under a multisig or governance layer. To achieve this, it is possible for the initial admin to set up all the required permissions, targets and functions, assign a new admin and finally renouncing to the admin role.
+
+Finally, the manager comes with a configurable `close` mode. When enabled by admins, it restricts every call to any managed contract while keeping permissions intact.
+
+```javascriot
+// Token's `restricted` functions closed
+await manager.setTargetClosed(myToken.address, true);
+
+// Token's `restricted` functions open
+await manager.setTargetClosed(myToken.address, false);
+```
+
+WARNING: Consider that even if a manager defines permissions for a contract function. These won't be applied if the managed contract instance is not using the xref:api:access.adoc#AccessManageddf-restricted--[`restricted`] modifier or if its manager is a different one.
+
+=== Role Admins and Guardians
+
+An important aspect of the AccessControl contract is that roles aren't granted nor revoked by role members. Instead, it relies on the concept of a role admin for granting and revoking.
+
+In the case of the AccessMananger, the same rule applies and only each role admin is able to call xref:api:access.adoc#AccessManager-grantRole-uint64-address-uint32-[grant] and xref:api:access.doc#AccessManager-revokeRole-uint64-address-[revoke] functions. Note that calling these functions will be subject to the execution delay the executing role admin has.
+
+Additionally, the access manager stores a _guardian_ an extra protection for each role and it is given the ability of canceling operations that have been scheduled by any role member with an execution delay. Consider that a role will have its initial admin and guardian default to the `ADMIN_ROLE` (`0`).
+
+IMPORTANT: Be careful with the `ADMIN_ROLE` members, since it acts as the default admin and guardian for every role, even a misbehaved guardian can cancel operations at discretion, affecting the manager operation.
+
+=== Manager configuration
+
+The manager contract provides a built-in interface for configuring permission settings. These functions can be accessed for those addresses member of the `ADMIN_ROLE`, which is constantly defined as `0` for all managers.
```

Comment:
I'm not sure I understand that sentence. It may be better to split it into multiple simpler ones.

Reply:
I rewrote it to:

> The manager contract provides a built-in interface for configuring permission settings that can be accessed by the manager's `ADMIN_ROLE` members.

Reply:
I still tink its confusing. 

Are you trying to say:
> Members of the `ADMIN_ROLE` can also configure additional settings.

Reply:
> provides a built-in interface

That part I'm really not sure about. I'm not sure what you are trying to say. What is a "built-in interface" ?

Reply:
> What is a "built-in interface" ?

Thanks for concretely pointing out what's confusing.
What I'm trying to convey is that the manager has a set of functions to manage permissions. Instead of managing timelocks separately to roles in different contracts, the manager's interface is enough.